extensions/community/CrazyGamesAdApi.json

```diff
@@ -0,0 +1,393 @@
+      "fullName": "Check if Ad Finished playing",
+      "functionType": "Condition",
+      "group": "Crazy Games",
+      "name": "AdFinishedPlaying",
+      "sentence": "Ad is finished playing?",
+      "events": [
+        {
+          "type": "BuiltinCommonInstructions::Standard",
+          "conditions": [],
+          "actions": [
+            {
+              "type": {
+                "value": "SetReturnBoolean"
+              },
+              "parameters": [
+                ""
+              ]
+            }
+          ]
+        },
+        {
+          "type": "BuiltinCommonInstructions::Standard",
+          "conditions": [
+            {
+              "type": {
+                "value": "SceneVariableAsBoolean"
+              },
+              "parameters": [
+                "__CrazyGamesExtension.adFinished",
+                "True"
+              ]
+            }
+          ],
+          "actions": [
+            {
+              "type": {
+                "value": "SetReturnBoolean"
+              },
+              "parameters": [
+                "True"
+              ]
+            }
+          ]
+        }
+      ],
+      "parameters": [],
+      "objectGroups": []
+    },
+    {
+      "description": "Check if Ad had an error.",
+      "fullName": "Check if Ad had an error",
+      "functionType": "Condition",
+      "group": "Crazy Games",
+      "name": "AdHadAnError",
+      "sentence": "Ad had an error?",
+      "events": [
+        {
+          "type": "BuiltinCommonInstructions::Standard",
+          "conditions": [],
+          "actions": [
+            {
+              "type": {
+                "value": "SetReturnBoolean"
+              },
+              "parameters": [
+                ""
+              ]
+            }
+          ]
+        },
+        {
+          "type": "BuiltinCommonInstructions::Standard",
+          "conditions": [
+            {
+              "type": {
+                "value": "SceneVariableAsBoolean"
+              },
+              "parameters": [
+                "__CrazyGamesExtension.adError",
+                "True"
+              ]
+            }
+          ],
+          "actions": [
+            {
+              "type": {
+                "value": "SetReturnBoolean"
+              },
+              "parameters": [
+                "True"
+              ]
+            }
+          ]
+        }
+      ],
+      "parameters": [],
+      "objectGroups": []
+    },
+    {
+      "description": "Returns an invite link.",
+      "fullName": "Returns an invite link",
+      "functionType": "ExpressionAndCondition",
+      "name": "InviteLink",
+      "sentence": "Returns an invite link _PARAM1_",
+      "events": [
+        {
+          "type": "BuiltinCommonInstructions::JsCode",
+          "inlineCode": [
+            "try {",
+            "    const crazysdk = window.CrazyGames.CrazySDK.getInstance();",
+            "    crazysdk.init();",
```

Comment:
You should probably not call crazysdk.init() in each action. Just do it once and save the fact that it's done in a boolean :)